### PR TITLE
Improve Headers design

### PR DESCRIPTION
**Pull request by B-Rass**

### CODE REVIEW

File: spec/components/page/heading_show_component_spec.rb [hunk 1]
```diff
@@ -14,12 +14,16 @@
...
     it "renders left_content with a back button" do # rubocop:disable RSpec/ExampleLength
       expect(rendered_component).to have_tag("div.col-12.bg-body") do
-        with_tag("div.d-flex") do
-          with_tag("a.btn-default.icon-link", title: "Retour", href: "http://test.host/sites") do
-            with_tag("span.bi-chevron-left")
-            with_tag("span.ms-2.d-none", text: "Retour")
+        with_tag("div.back-button-container") do
+          with_tag("a.btn.back-button") do
+            with_tag("i.bi-chevron-left")
+            with_tag("span.ms-2", text: "Retour")
```
Comment (nicolas-brousse) [inline spec/components/page/heading_show_component_spec.rb:20]: Maybe we could keep the check on the back url href?

File: spec/components/page/heading_new_component_spec.rb [hunk 2]
```diff
@@ -11,17 +11,17 @@
...
+  it "renders a back button" do # rubocop:disable RSpec/ExampleLength
     expect(rendered_component).to have_tag("div.col-12.bg-body") do
-      with_tag("div.d-flex") do
-        with_tag("span.flex-grow-1") do
-          with_tag("a.btn-default.icon-link", title: "Retour", href: "http://test.host/sites") do
-            with_tag("span.bi-chevron-left")
-            with_tag("span.ms-2.d-none", text: "Retour")
-          end
+      with_tag("div.back-button-container") do
+        with_tag("a.btn.back-button") do
```
Comment (nicolas-brousse) [inline spec/components/page/heading_new_component_spec.rb:17]: Maybe we could keep the check on the back url href?

File: spec/components/page/heading_edit_component_spec.rb [hunk 3]
```diff
@@ -12,14 +12,18 @@
...
   context "without extra_buttons" do
-    it "renders left_content with a back button" do # rubocop:disable RSpec/ExampleLength
+    it "renders heading with a back button" do # rubocop:disable RSpec/ExampleLength
       expect(rendered_component).to have_tag("div.col-12.bg-body") do
-        with_tag("div.d-flex") do
-          with_tag("a.btn-default.icon-link", title: "Retour", href: "http://test.host/sites") do
-            with_tag("span.bi-chevron-left")
-            with_tag("span.ms-2.d-none", text: "Retour")
+        with_tag("div.back-button-container") do
+          with_tag("a.btn.back-button") do
```
Comment (nicolas-brousse) [inline spec/components/page/heading_edit_component_spec.rb:18]: Maybe we could keep the check on the back url href?

File: spec/components/page/heading_component_spec.rb [hunk 4]
```diff
@@ -32,40 +33,47 @@
...
-    it "renders heading" do
-      expect(rendered_component).to have_tag("div.col-12") do
-        with_tag("a.btn", text: "Button")
+  context "with back_button_url" do
+    let(:back_button_url) { "url" }
+
+    it "renders heading" do # rubocop:disable RSpec/ExampleLength
+      expect(rendered_component).to have_tag("div.col-12.bg-body") do
+        with_tag("div.back-button-container") do
+          with_tag("a.btn.back-button") do
```
Comment (nicolas-brousse) [inline spec/components/page/heading_component_spec.rb:58]: Maybe we could check the href value?

File: spec/components/page/heading_edit_component_spec.rb (hunk 3 above)
Comment (B-Rass) [reply spec/components/page/heading_edit_component_spec.rb:18]: Yes you're right

**Review by nicolas-brousse — APPROVED** [review]

**Pull request merged**